proto/tss/tofnd/v1beta1/common.proto

```diff
@@ -8,7 +8,10 @@ import "gogoproto/gogo.proto";
 package tss.tofnd.v1beta1;
 
 // Key presence check types
-message KeyPresenceRequest { string key_uid = 1; }
+message KeyPresenceRequest {
+  string key_uid = 1;
+  bytes pub_key = 2;
```

Comment:
why do we need the pubkey here? is the id not unique anymore?

Reply:
Yeah, tofnd doesn't store any information about key ID to private key mapping. Since private keys were a function of the static mnemonic and key ID, this was ok before. But now the key id isn't sufficient to distinguish which of the tofnd mnemonics is to be used. So, we figure that out using the pub key.